### PR TITLE
Improved changelogs for readability and understanding

### PR DESCRIPTION
**Pull request by jessefriedman**

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -78,24 +78,32 @@ There are opportunities for developers at all levels to contribute. [Learn more
...
-* Widgets: they're now compatible with WP 4.5 Customizer's Selective Refresh. This allows to update a widget and, instead of reloading the entire page in preview, only its portion in preview is refreshed.
-* Widget Visibility: new options available to choose to hide or show a widget in custom post type single and archive views.
+* Protect: the routine that verifies your site is protected from brute-force attacks got some love and is more efficient.
+* Contact Forms: cleaning the database of spam form submission records is more efficient. Nothing to see here!
+
+Feature Improvements:
+
+* VideoPress: edit your VideoPress shortcode in the editor with a fancy new modal options window.
+* Custom Content Types are now classier: a new CSS class on Testimonial featured images — has-testimonial-thumbnail — allows you to customize Jetpack custom post types as you see fit.
+* Sharing: social icons are now placed under “the add to cart” singular product views in WooCommerce, making it easier for customers to share their purchases on social media.
```
Comment (dereksmart) [inline readme.txt]: I think the only part you meant to put in quotes was `"add to cart"`

File: readme.txt [hunk 2]
```diff
@@ -78,24 +78,32 @@ There are opportunities for developers at all levels to contribute. [Learn more
...
+* Theme Tools: schema.org microdata makes breadcrumbs tastier with improved schema.org microdata. Google will now have an easier time knowing what page it’s on, and how that page relates to the other pages in your site hierarchy. SEO, ftw.
+* Widget Visibility: now you can select widgets and when to show or hide them right from custom post type single and archive views.
+
+Jetpack UI Improvements:
+
+* What’s in it for me? We’ve done a better job explaining the benefits of Jetpack and connecting it to WordPress.com.
+* Shortcodes: handy links to shortcode documentation convey the types of media you can quickly and safely embed.
+* Widgets: Jetpack widgets now refresh in the customizer without making you refresh the entire page. Live previews, yes indeed.
```
Comment (dereksmart) [inline readme.txt]: Maybe mention this is only for >= 4.5 WP

File: readme.txt [hunk 3]
```diff
@@ -78,24 +78,32 @@ There are opportunities for developers at all levels to contribute. [Learn more
...
+
+Bug Fixes:
+
+* Comments: we fixed a mistake where a comment subscription checkbox appeared on custom post types — despite the fact you couldn’t actually subscribe to those types of comments. Our bad.
+* Contact Forms: we fixed a bug where the telephone field (which can only be added manually) rendered incorrectly — breaking some forms in the process.
+* General: we blocked direct access to the Jetpack_IXR_Client class which caused fatal PHP errors in some server setups.
+* Shortcodes: we removed the frameborder attribute in the YouTube embed code. It was deprecated in HTML 5. Newly retired, the frameborder attribute can now be seen mall walking with its pants pulled up to its armpits.
```
Comment (dereksmart) [inline readme.txt]: *at the mall.  

Also, I don't get the joke?

Comment (dereksmart) [inline readme.txt]: Oh.  newly retired.

File: readme.txt [hunk 4]
```diff
@@ -78,24 +78,32 @@ There are opportunities for developers at all levels to contribute. [Learn more
...
-* Widgets: they're now compatible with WP 4.5 Customizer's Selective Refresh. This allows to update a widget and, instead of reloading the entire page in preview, only its portion in preview is refreshed.
-* Widget Visibility: new options available to choose to hide or show a widget in custom post type single and archive views.
+* Protect: the routine that verifies your site is protected from brute-force attacks got some love and is more efficient.
+* Contact Forms: cleaning the database of spam form submission records is more efficient. Nothing to see here!
+
+Feature Improvements:
+
+* VideoPress: edit your VideoPress shortcode in the editor with a fancy new modal options window.
+* Custom Content Types are now classier: a new CSS class on Testimonial featured images — has-testimonial-thumbnail — allows you to customize Jetpack custom post types as you see fit.
+* Sharing: social icons are now placed under the "add to cart” singular product views in WooCommerce, making it easier for customers to share their purchases on social media.
```
Comment (eliorivero) [inline readme.txt]: The "share their purchases" phrase might not be entirely correct since the product someone is sharing might not have been purchased.

**Pull request merged**